portfolio/src/main/webapp/script.js

```diff
@@ -92,3 +92,11 @@ window.addEventListener('scroll', function() {
     document.getElementById('logo').style.fontSize = '35px';
   }
 })
+
+/**
+ */
+function getMessage() {
+  fetch('/data').then(response => response.text()).then((message) => {
```

Comment:
This is essentially non-functional until the servlet is included, right? In cases that code heavily depends on each other, it's best to include them simultaneously in the Pull Request. It's better to do this because you don't want your branch Master to be in a broken state (right now I think the button press that calls this method would be broken).

Reply:
I realized that this is untrue, because the Servlet already exists (my bad!) and is registered as 'data', so this should be fine! Forget I said anything :)

Reply:
Good! :)